src/emulator/auth/operations.ts

```diff
@@ -2830,6 +2904,246 @@ function updateTenant(
   return state.updateTenant(reqBody, ctx.params.query.updateMask);
 }
 
+// TODO: Timeout is 60s. Should we make the timeout an emulator configuration?
+async function fetchBlockingFunctionWithTimeoutAndUpdateUser(
```

Comment:
- Split logic for fetching from logic that applies modifiable fields
- Rename to something shorter, e.g. `fetchBlockingFunction`
- Generally will want to return update instead of having side effect